src/main/java/com/github/fge/jsonpatch/JsonPatch.java

```diff
@@ -123,11 +123,9 @@ public JsonPatch(final List<JsonPatchOperation> operations)
      * @throws NullPointerException input is null
      */
     public static JsonPatch fromJson(final JsonNode node)
-        throws IOException
-    {
+            throws IOException, JsonPatchException {
```

Comment:
Sounds good.  JsonProcessingException extends IOException, so that will be handled as well.